samples/containers/hello-back-button/src/main/java/com/squareup/sample/hellobackbutton/HelloBackButtonLayoutRunner.kt

```diff
@@ -2,6 +2,7 @@ package com.squareup.sample.hellobackbutton
 
 import android.view.View
 import android.widget.TextView
+import com.squareup.sample.hellobackbutton.R.layout
```

Comment:
Some day, they'll fix this in Android Studio.

Reply:
I'll take that bet.